stdlib/ballerina-http/src/main/ballerina/http/http_secure_client.bal

```diff
@@ -16,15 +16,17 @@
 
 
 import ballerina/io;
-import ballerina/runtime;
 import ballerina/mime;
+import ballerina/runtime;
 
 @final string EMPTY_STRING = "";
 @final string WHITE_SPACE = " ";
-@final string CONTENT_TYPE_HEADER = "Content-Type";
-@final string BASIC_SCHEME = "basic";
-@final string OAUTH_SCHEME = "oauth";
-@final string JWT_SCHEME = "jwt";
+
+public type AUTH_SCHEME "basic"|"oauth2"|"jwt";
+
+@final public AUTH_SCHEME BASIC = "basic";
+@final public AUTH_SCHEME OAUTH2 = "oauth2";
+@final public AUTH_SCHEME JWT = "jwt";
```

Comment:
Shall we rethink the names for these constants? The convention we follow for these sorta enums is that the names should be descriptive enough so that when reading the code, it's obvious to the reader what the constants are about. 

For example, when used in code, the above would be: `http:BASIC` 
Prefixing the above with say, `AUTH` for instance, it'd be: `http:AUTH_BASIC`

Reply:
Yes agree with that. But will that be okay for authentication schemes like **OAuth2**?
Then user will refer like `http:AUTH_OAUTH2`.

The complete use case for above example would be as follows:
```
authConfig: {
     scheme: http:AUTH_OAUTH2
}
```

Reply:
Shall we check this with the others? I think this might be ok given that OAuth2 is the scheme name in this instance

Reply:
Fixed according to the discussion had.